lib/eslint.js

```diff
@@ -413,6 +413,7 @@ function prepareConfig(config) {
         globals: util.mergeConfigs({}, config.globals),
         env: util.mergeConfigs({}, config.env || {}),
         settings: util.mergeConfigs({}, config.settings || {}),
+        disableInlineConfig: config.disableInlineConfig || false,
```

Comment:
Ah, that makes sense actually. As long as I can enable it via the CLI and using CLIEngine I’m happy; honestly I was a bit lost on where to add things and may have went a bit trigger-happy.